doc/Installation.md

```diff
@@ -169,14 +169,13 @@ user these additional privileges:
 
 For reference or if you want to create the table schema for yourself to avoid
 having to give PrivateBin too many permissions (replace `prefix_` with your own
-table prefix and create the table schema with your favourite MariaDB/MySQL
+table prefix and create the table schema with your favorite MariaDB/MySQL
 client):
 
 ```sql
 CREATE TABLE prefix_paste (
     dataid CHAR(16) NOT NULL,
     data MEDIUMBLOB,
-    postdate INT,
```

Comment:
Unlikely to break anything, but we have no (integration) tests that verify instances with the old DB schema still work with new PrivateBin?

Reply:
Yes we do. Hence I had to update a number of test cases as well. Thanks to these upgrade tests I discovered that not all SQLite versions support "DROP COLUMN" and added that version check (the version used with PHP in my unit test container and on Github do support it, but Scrutinizer uses an older one).
https://github.com/PrivateBin/PrivateBin/blob/ec02afca0469017a77fa5c8b952241c879883882/tst/Data/DatabaseTest.php#L326

We are limited by only testing SQLite but not on MariaDB, MySQL, Postgres, etc. If we wanted to test on those, we could extend the ControllerTest class, like we do for SQLite and keep those tests disabled by default, but for example on some dedicated Github actions, set up MariaDB & PG environments and enable these tests to run there.